zfmt/ast.go

```diff
@@ -518,6 +518,21 @@ func (c *canon) proc(p ast.Op) {
 			c.space()
 			c.exprs(p.Args)
 		}
+	case *ast.Load:
+		c.next()
+		c.write("load %s", zson.QuotedString([]byte(p.Pool)))
+		if p.Branch != "" {
+			c.write(p.Branch)
+		}
+		if p.Author != "" {
+			c.write(" author %s ", p.Author)
```

Comment:
We want just a leading space on these so they'll look good regardless of which are set.
```suggestion
			c.write(" author %s", p.Author)
```